setup.cfg

```diff
@@ -0,0 +1,5 @@
+[flake8]
+max-complexity = 12
+ignore = E203, E231, E501, E722, W503, B950
```

Comment:
That is also fairly subjective, better use defaults?

Reply:
I think (IIRC) that these are listed by black as broken flake8 rules that need to be turned off since they are covered by Black, and will sometimes trigger on valid PEP 8 code. I could be wrong though, I've been sort of carrying them along.

Reply:
You can pick these up from the page [here](https://black.readthedocs.io/en/stable/the_black_code_style.html), somewhat sprinkled around the page. I may have 2-3 too many here.

Reply:
TODO:when cleaning this up tomorrow, also remove max-complexity (oops!).

Reply:
All are black related now. Wow, have no idea how `E722` snuck in (bare except). Fixed.